test/integration/integration_test.go

```diff
@@ -245,7 +246,13 @@ func FeatureContext(s *godog.Suite) {
 
 	s.BeforeSuite(func() {
 		testDir = setUp()
-		util.StartLog(testDir)
+		testResultDir = filepath.Join(testDir, "..", "test-results")
```

Comment:
That means now it not going to make that `test-results` in the testDir but one more level up.

Reply:
@praveenkumar Yes, it is one level up. Now content of `out` directory is:
```
bindata
integration-test
test-results
```